app/views/conferences/_conference_details.html.haml

```diff
@@ -30,6 +30,10 @@
                 = link_to "Register", new_conference_conference_registration_path(conference.short_title), class: "btn btn-default", disabled: cannot?(:new, Registration.new(conference_id: conference.id))
                 - if cannot?(:new, Registration.new(conference_id: conference.id)) && conference.registration_limit_exceeded?
                   Sorry, no places left
+            - if !current_user.nil? && current_user.tracks.where(program: conference.program).length > 0
+              = link_to "My Track Requests", conference_program_tracks_path(conference.short_title), class: 'btn btn-default'
+            - elsif can? :new, conference.program.tracks.new
+              = link_to "Submit Track Request", new_conference_program_track_path(conference.short_title), class: 'btn btn-default'
```

Comment:
haml-lint doesn't have a linter that checks for single or double quotes https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md, but it should have caught it through its [rubocop integration](https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#rubocop). I made some tries and failed to make it catch it